QUARKUS-1026.md

```diff
@@ -0,0 +1,33 @@
+# QUARKUS-1026 Continuous Testing in Dev Mode
+
+JIRA link: https://issues.redhat.com/browse/QUARKUS-1026
+
+Test plan covers Continuous Testing in Dev Mode functionality: https://quarkus.io/guides/continuous-testing
+
+## Scope of the testing
+- Enable/Disable Continuous Testing on a running Quarkus application
+- Cover DEV UI functionality about continuous testing
+- Detect new tests when continuous testing is enabled
+- Run affected tests after new changes when continuous testing is enabled
+- Dev services should be deployed/undeployed for tests (if required) when continuous testing is enabled
+ - Kafka Dev services
```

Comment:
To consider:
AMQP Dev services https://quarkus.io/guides/amqp-dev-services

Reply:
Added